### ISSUE TITLE
new release?

### ISSUE THREAD

**Issue by bldewolf** [FreeRADIUS/freeradius-client#83]
Hi, I use the check_radius plugin from nagios-plugins that uses freeradius-client for running its RADIUS checks.  I was looking in to updating the plugin to support IPv6 and noticed that, while freeradius-client has patches for IPv6 support, they aren't in any released version.  Are there plans to make a new release?

**Comment by alandekok** [comment]
We should be doing a new release soon.

**Comment by exharrison** [comment]
Alan,
Any update? :)

Thanks!

**Comment by exharrison** [comment]
Hey, just checking in again to see if there is any ETA for a new release.

Thanks!

**Comment by exharrison** [comment]
Hey Just checking in again if there is any information on a new release.
Thanks

**Comment by mcnewton** [comment]
Released this week.

**Comment by mcnewton** [comment]
Oops... freeradius-server, not freeradius-client, disregard that.

**Comment by exharrison** [comment]
Is there a freeradius-client release coming behind the freeradius-server?

**Comment by exharrison** [comment]
Any plans on an official 1.1.8 release?

**Comment by exharrison** [comment]
Just checking in to see if an official 1.1.8 release has been scheduled.

**Comment by CHITHINH** [comment]
I have checked, it seems the EAP support PR started since 2016 and is still in progress. Is there any plan for this support, will it releas in 1.1.8?

Thanks

**Comment by johnnybubonic** [comment]
Just giving a poke; RHEL-based systems as of 8.x, CentOS Stream, et. al. have dropped radiusclient-ng and are now using freeradius-client (1.1.7). However, due to #42, this is causing issues for me. #46 fixes but is not in 1.1.7.

Any idea on when 1.1.8 will release?

**Comment by alandekok** [comment]
I've just released 1.1.8.  Please let me know if there are any issues.

**Comment by alandekok** [comment]
@CHITHINH We're not planning on adding EAP support.  That is an enormous amount of work.  I would suggest using `eapol_test` from the `hostap` distribution.

**Comment by CHITHINH** [comment]
> @CHITHINH We're not planning on adding EAP support. That is an enormous amount of work. I would suggest using `eapol_test` from the `hostap` distribution.

Thanks for the confirmation.

**Comment by horaceou** [comment]
> I've just released 1.1.8. Please let me know if there are any issues.

Hi Alan, I recently used freeradius-client 1.1.8 on Linux with freeradius-server 3.0.25 on Ubuntu.
Before testing I have enabled the server by :

**_sudo freeradius -X_**

to listen the request from freeradius-client. 

Then, I tried below command on Linux to log in :

**_radiusclient -f /etc/radiusclient/radiusclient.conf User-Name=bruno Password=boss123_**

But no response log on server side. BTW, I also used radlogin command to test the authentication :

**_radlogin -f /etc/radiusclient/radiusclient.conf
login : bruno
Password : boss123_**

But the freeradius-server had no response, either. 

Is there any ideas of this issue ? Thank you.

PS. The radius authentication can work fine on version 1.1.7.
(modified radiusclient.conf and server can be seen in [https://github.com/horaceou/freeradius-client](url) )
